### ISSUE TITLE
day01/ex05

### ISSUE THREAD

**Issue by Ibouch** [42-AI/bootcamp_python#99]
* Day: 01
* Exercise: 05

The example of the `Account` class with the two banking_test files produces this message :
```python
TypeError: __init__() takes 2 positional arguments but 3 were given
```

Maybe replace
```python
bank.add(Account('William John', {'zip': '100-064', 'value': 6460.0, 'ref': '58ba2b9954cd278eda8a84147ca73c87', 'info': None, 'other': 'This is the vice president of the corporation'}))
```
by something like that
```python
bank.add(Account(
    'William John',
    zip='100-064',
    value=6460.0,
    ref='58ba2b9954cd278eda8a84147ca73c87',
    info=None,
    other='This is the vice president of the corporation'
))
```

**Comment by matboivin** [comment]
@Ibouch Thanks a lot! :smile: I am going to notify other members about your PR.

**Comment by matboivin** [comment]
Fixed by #100
